addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -610,22 +664,31 @@ async function toInline($editable, cssRules, $iframe) {
     enforceImagesResponsivity(editable);
     await flattenBackgroundImages(editable);
 
-    // Remove contenteditable attributes
-    [editable, ...editable.querySelectorAll('[contenteditable]')].forEach(node => node.removeAttribute('contenteditable'));
-
-    // Hide replaced cells on Outlook
-    for (const toHide of editable.querySelectorAll('.mso-hide')) {
-        const style = toHide.getAttribute('style') || '';
-        toHide.setAttribute('style', `${style} mso-hide: all;`.trim());
-    }
-
-    // Styles were applied inline, we don't need a style element anymore.
-    $editable.find('style').remove();
-
-    for (const [node, displayValue] of displaysToRestore) {
-        node.style.setProperty('display', displayValue);
-    }
-    $editable.addClass('odoo-editor-editable');
+    toInlineWrapUp($editable, displaysToRestore);
+}
+/**
+ * Convert the contents of an editable area (as a JQuery element) into it's
+ * inline version. If no CSS Rules are given, they
+ * will be computed for the editable element's owner document.
+ *
+ * @param {JQuery} $editable
+ * @param {Object[]} [cssRules] Array<{selector: string;
+ *                                   style: {[styleName]: string};
+ *                                   specificity: number;}>
+ * @param {JQuery} [$iframe] the iframe containing the editable, if any
+ */
+function simpleToInline($editable, cssRules, $iframe) {
```

Comment:
I think it's confusing to have `toInline` and `simpleToInline`. I mean: how would someone be able to know which one is the one to use in their particular case ? Correct me if I'm wrong but I feel like this is basically a copy paste of `toInline` with just some parts of the code that are _not_ executed compared to the original `toInline`. I feel like it would be less confusing to have a single entry point, `toInline`, with options to either enable extra inlining or disable some of the inlining. Like for example there are stuff related to boostrap so we could have `bootstrap: false`, or it seems like things related to tables seem to have been left out so we could have`tables: false` or something like that. What do you think ? It would at least have the benefit of not having to externalize some part of the code in vague subfunctions like "setup" or "wrapup".

In addition, it is not clear to me in fact why we need to _not_ call those conversion functions. I mean like the `bootstrapToTable` or `cardToTable` for example. If they don't match the content inside the email they won't do anything anyway, right ? And in the case where they would match the content then we need them otherwise the mail is not going to be displayed properly, correct ?

I feel like we should do a thorough analysis of what is actually going to create issues and what isn't as it isn't clear in my mind what are the problematic cases exactly and I feel like this function here is exagerating on the cautious side of things at the expense of adding quite the complexity around these parts of the code.